<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QWidget" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="CheckCommodity">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>查看所有商品</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pull">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>下架商品</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CheckOrders">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>查看所有订单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Check_users">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>20</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>查看所有用户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BanUser">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>20</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>封禁用户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>20</y>
     <width>155</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>1221</width>
     <height>601</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="search">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>100</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>1071</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
